openshift/resolve-yamls.sh

```diff
@@ -24,7 +24,7 @@ function resolve_resources() {
                            sed -e 's/ /|/g' -e 's/^/(/' -e 's/|$/)\n/')
 
   >$resolved_file_name
-  for yaml in $(find $dir -name "*.yaml" | grep -vE $ignores); do
+  for yaml in $(find $dir -name "*.yaml" | grep -vE $ignores | sort); do
```

Comment:
I think  `find -s ` may be better here than `sort`

Reply:
this change is already in there https://github.com/openshift/tektoncd-pipeline/blob/953d50db65a8e4967b9a848952e1756990681420/openshift/resolve-yamls.sh#L27

@sthaha as we discussed during 0.7.0 release